app/workers/form1095/get_new1095_bs.rb

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
+
+require 'sidekiq-scheduler'
+
+module Form1095
+  class GetNew1095Bs
```

Comment:
Avoid using `Get` in class names and end worker classes with `Job`.
```suggestion
  class New1095BsJob
```